pkg/models/model.go

```diff
@@ -123,9 +131,33 @@ func (m *Model) setNodeMap() error {
 				NetLayers:     netLayers,
 				CapacityBlock: cb.Name,
 			}
+
+			region, ok := metadata["region"]
+			if !ok {
+				region = "none"
+			}
+			if _, ok := regions[region]; !ok {
+				regions[region] = make(map[string]string)
+
+			}
+			regions[region][name] = name
 		}
 	}
 
+	regionNames := make([]string, 0, len(regions))
+	for region := range regions {
+		regionNames = append(regionNames, region)
+	}
```

Comment:
Nit: You could you use `copy` here instead, right?

```suggestion
	regionNames := make([]string, len(regions))
	copy(regionNames, regions)
```

Reply:
regions is a map, regionNames is an array